e2e-tests/portal/cypress/e2e/journeys/maker-about-supply-contract/fill-in-about-buyer-page-happy-path.spec.js

```diff
@@ -1,28 +1,22 @@
-const { contract, contractAboutSupplier, contractAboutBuyer, contractAboutPreview, defaults } = require('../../pages');
+const { contract, contractAboutSupplier, contractAboutBuyer, contractAboutFinancial, defaults } = require('../../pages');
 const partials = require('../../partials');
 const MOCK_USERS = require('../../../../../e2e-fixtures');
-const aDealWithAboutSupplyContractComplete = require('./dealWithFirstPageComplete.json');
+const { additionalRefName } = require('../../../fixtures/deal');
 
 const { BANK1_MAKER1 } = MOCK_USERS;
 
 context('about-supply-contract', () => {
-  let deal;
-
   before(() => {
-    cy.insertOneDeal(aDealWithAboutSupplyContractComplete, BANK1_MAKER1).then((insertedDeal) => {
-      deal = insertedDeal;
-    });
+    cy.createBssEwcsDeal();
   });
 
-  it('A maker picks up a deal with the supplier details completed, and fills in the about-buyer-contract section, using the companies house search.', () => {
-    cy.login(BANK1_MAKER1);
+  it('A maker picks up a deal and fills in the about-buyer-contract section, using the companies house search.', () => {
+    cy.loginGoToDealPage(BANK1_MAKER1);
 
-    // navigate to the about-buyer page
-    contract.visit(deal);
     contract.aboutSupplierDetailsLink().click();
     contractAboutSupplier.nextPage().click();
 
-    cy.title().should('eq', `Buyer information - ${deal.additionalRefName}${defaults.pageTitleAppend}`);
+    cy.assert('title', `Buyer information - ${additionalRefName}${defaults.pageTitleAppend}`);
```

Comment:
should this not be 
```suggestion
    cy.assert(cy.title(), `Buyer information - ${additionalRefName}${defaults.pageTitleAppend}`);
```
?

Reply:
Changed to assertText